File: src/client/src/FantomasTools/Loader.fs
module FantomasTools.Client.Loader

open Fable.React
open Fable.React.Props
open Reactstrap

let loader =
    div [ ClassName "loader" ] [
        div [ ClassName "inner" ] [
            Spinner.spinner [ Spinner.Color Primary ] []
        ]
    ]

File: _dist_/bin/Loader.js
import * as react from "../../web_modules/react.js";
import { SpinnerProps, spinner } from "./.fable/Fable.Reactstrap.0.5.1/Spinner.fs.js";

export const loader = react.createElement("div", {
    className: "loader",
}, react.createElement("div", {
    className: "inner",
}, spinner([new SpinnerProps(2, "primary")], [])));

